src/backend/src/prisma/schema.prisma

```diff
@@ -117,6 +117,14 @@ model User {
   archivedTeams             Team[]                  @relation(name: "userArchived")
   createdMaterialTypes      Material_Type[]         @relation(name: "materialTypeCreator")
   createdManufacturers      Manufacturer[]          @relation(name: "manufacturerCreator")
+  requiredDesignReviews     Design_Review[]         @relation(name: "requiredAttendee")
+  optionalDesignReviews     Design_Review[]         @relation(name: "optionalAttendee")
+  confirmedDesignReviews    Design_Review[]         @relation(name: "confirmedAttendee")
```

Comment:
call this userConfirmedDesignReviews maybe, this name suggests, that the design review itself has been confirmed/ finalized